src/displaySinglePokemon.js

```diff
@@ -0,0 +1,162 @@
+import get from './getElement.js';
+
+const displayPokemon = (data) => {
+  const pokemon = data;
+  const name = pokemon.name;
+  const image = pokemon.sprites.front_default;
```

Comment:
You can create a mapper function to map the received data to the data that you need.